### PR TITLE
Opening a support case for Trento

### PR DESCRIPTION
**Pull request by abravosuse**
Adding a section explaining how to open a support case for Trento in the SUSE Customer Center.

### PR creator: Description

Describe the overall goals of this pull request.

### PR creator: Are there any relevant issues/feature requests?

* bsc#...
* jsc#SLE-...

### CODE REVIEW

**Review by tomschr — CHANGES_REQUESTED** [review]
@abravosuse Thanks Alberto, much appreciated! :+1:  I have some smaller issues.

File: trento/xml/article_sap_trento.xml [hunk 1]
```diff
@@ -1565,6 +1565,46 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
...
+    </para>
+      <para>
+      When opening a support case for Trento, make sure that you attach the following information:
+    </para>
+         <itemizedlist>
+          <listitem>
+            <para>The output of the Trento support plugin.</para>
+          </listitem>
+           <listitem>
+            <para>A dump of your scenario.</para>
```
[the remark this answers is not in the document]
Comment (tomschr) [reply trento/xml/article_sap_trento.xml:1583]: Ahh right. If you only review things in isolation, such things can be missed. Thanks for the hint! :+1: 

Is "scenario dump" a real technical term used upstream etc?

[the remark this answers is not in the document]
Comment (abravosuse) [reply trento/xml/article_sap_trento.xml:1583]: That is how the Trento core team refers to it: https://github.com/trento-project/web/blob/5cee0bf5e3b277dd601b57ca5dc9905e4fe30287/hack/dump_scenario_from_k8.sh

Why adding a pointer for the scenario dump but not for the trento support plugin?

Instead I suggest that we move the last paragraph in the section, the one that refers to the next section, before listing the content to be provided. That way is clear where they can find information on how to retrieve the required information.

File: trento/xml/article_sap_trento.xml [hunk 2]
```diff
@@ -1565,6 +1565,46 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
...
+          </listitem>
+          <listitem>
+            <para>The output of the supportconfig command in the Agent host. See
+             https://documentation.suse.com/sles/15-SP1/html/SLES-all/cha-adm-support.html#sec-admsupport-cli
+             for information on how to run this command from Command Line.
+           </para>
+          </listitem>
+        </itemizedlist>
+      <para>
+      Section <xref linkend="sec-trento-problemanalysis" /> explains how to retrieve the information listed above.
```
[the remark this answers is not in the document]
Comment (abravosuse) [reply trento/xml/article_sap_trento.xml:1604]: That's what I was referring to in the last part of my previous comment. But I wouldn't move it to the list item in line 183. I would move it before the itemized list, as it relates to all the points in the list.

File: trento/xml/article_sap_trento.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (tomschr) [reply trento/xml/article_sap_trento.xml:1583]: Should work too. Sounds good! :+1:

[the remark this answers is not in the document]
Comment (abravosuse) [reply trento/xml/article_sap_trento.xml:1584]: My latest commit collects this change. Please review.

[the remark this answers is not in the document]
Comment (tomschr) [reply trento/xml/article_sap_trento.xml:1584]: Thanks Alberto. :+1:  I think it's fine now. If you are finished, I can merge it.

[the remark this answers is not in the document]
Comment (abravosuse) [reply trento/xml/article_sap_trento.xml:1584]: Yes, I am done. You can merge it!

**Pull request merged**